src/core.c/IO/Path.pm6

```diff
@@ -377,9 +377,9 @@ my class IO::Path is Cool does IO {
           $!SPEC.join('', $!path, child.Str)
     }
 
-    method add (IO::Path:D: \child) {
+    method add (IO::Path:D: *@children) {
```

Comment:
Wonder if it's worth a pair of `multi`s given the common case is just one child to add, and we can save an amount of work?